### PR TITLE
Update start.ipynb

### PR DESCRIPTION
**Pull request by senecameeks**
Update the start tutorial with a new notebook that details how to run circuits using our hardware. Note, only some people will have access to use our physical devices.

### CODE REVIEW

File: docs/tutorials/google/start.ipynb [hunk 1]
```diff
@@ -1,478 +1,532 @@
 {
```
Comment (wcourtney) [inline docs/tutorials/google/start.ipynb:1]: <p>s/medated/mediated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (wcourtney) [inline docs/tutorials/google/start.ipynb:1]: <p>Update "best practices" link to <a href="https://quantumai.google/cirq/google/best_practices" target="_blank">https://quantumai.google/cirq/google/best_practices</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (wcourtney) [inline docs/tutorials/google/start.ipynb:1]: <p>s/credendtials/credentials</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Comment by wcourtney** [comment]
Added a few quick-to-resolve typo and link fixes. Otherwise LGTM to publish before resolving other issues in go/quacstart-feedback so that we have something posted.

**Review by wcourtney — APPROVED** [review]
LGTM, thanks!

File: docs/tutorials/google/start.ipynb (hunk 1 above)
Comment (aasfaw) [inline docs/tutorials/google/start.ipynb:1]: <p>s/After running the block you will see/After running the block, you will see</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (aasfaw) [inline docs/tutorials/google/start.ipynb:1]: <p>First, decide with which Google Cloud project you will use the Quantum Research Platform. All of your quantum programs and results will live under a project which you specify when creating and running these programs using Quantum Engine. You can use an existing project or create a new project.&nbsp;<a href="https://cloud.google.com/docs/overview/" target="_blank">Learn more about creating a project</a>.  --&gt; First, decide whether you want to <a href="https://cloud.google.com/docs/overview/" target="_blank">create a new Google Cloud Platform project</a>, or use an existing one.</p><p> </p><p>Remove bullet that says "Log in and agree to&nbsp;<a href="https://cloud.google.com/terms/" target="_blank">Terms of Service</a>." It's implied.</p><p> </p><p>Remove "You can run the code by either clicking the play button (pointed by arrow below) or by selecting the block and pressing CTRL-ENTER." </p><p> </p><p>Remove "After you authenticate and allow access to your project, you will be given a string which you should enter into the text box that appears in the run area (and then press return)." -- this is already implied in "follow the authentication flow"</p><p> </p><p>Replace "If this fails, make sure that you have cut and paste the string correctly (e.g. the clipboard button seems to not work for some browser/OS combinations)." and replace to "Contact your partner in the Quantum Hardware Residency Program if you encounter issues at this stage."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (aasfaw) [inline docs/tutorials/google/start.ipynb:1]: <p>s/Now that you've enabled Quantum Computing Service and configured the notebook, let's/Now, let's</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (aasfaw) [inline docs/tutorials/google/start.ipynb:1]: <p>Add another text block followed by code.</p><p> </p><p>Text: "If your quantum processor is calibrated to do a specific set of gates, you will need to transform your circuit into that gateset. For example, for a CZ gateset, you can do the following." </p><p> </p><p>Code:</p><p> </p><pre class="ql-syntax" spellcheck="false">gateset = cirq.CZTargetGateset(allow_partial_czs=True)
hw_circuit = cirq.optimize_for_target_gateset(hw_circuit, gateset=gateset)
print(hw_circuit)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (aasfaw) [reply docs/tutorials/google/start.ipynb:1]: <p>Add to that text above: Other gatesets and considerations are included in the best practices guide</p><p> </p><p>with same hyperlink as Will Courtney's above.</p>

Comment (aasfaw) [reply docs/tutorials/google/start.ipynb:1]: <p>replace </p><p> </p><p>Google's Quantum Research Platform enables researchers to run quantum programs on Google's quantum processors. Access is mediated through the Quantum Engine service. </p><p> </p><p>with</p><p> </p><p>Access to Google's quantum processors is mediated through the Quantum Engine service.</p>

Comment (aasfaw) [inline docs/tutorials/google/start.ipynb:1]: <h1>Getting set up to use Google's quantum processors</h1><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (senecameeks) [reply docs/tutorials/google/start.ipynb:1]: Will follow up with another PR to add this improvement. Up to this point I've been editing the codelab's raw json file directly (this was easiest for the initial quick changes), but adding another text/code block this way will likely cause errors.

Comment (senecameeks) [reply docs/tutorials/google/start.ipynb:1]: The followup PR is #6346

**Pull request merged**